src/libsystemd-network/dhcp-option.c

```diff
@@ -48,6 +49,32 @@ static int option_append(uint8_t options[], size_t size, size_t *offset,
                 *offset += 1;
                 break;
 
+        case SD_DHCP_OPTION_USER_CLASS: {
+                unsigned i = 0;
```

Comment:
`i` is a surprising name for an accumulator we add up byte sizes in. Also, I figure we should use "size_t" since this is about summing up memory sizes.